Services/BrokerService/Controllers/BrokersController.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System.Collections.Generic;
+using Microsoft.AspNetCore.Mvc;
+using BrokerService.Interfaces;
+using DTO;
+
+namespace BrokerService.Controllers
+{
+    [ApiController]
+    [Route("[controller]")]
+    public class BrokersController : ControllerBase
+    {
+        [Route("instruments/get")]
+        [HttpGet]
+        public IEnumerable<Instrument> GetInstruments(
+                [FromServices] IGetInstrumentsCommand command,
+                [FromQuery] BankType bank,
```

Comment:
Maybe replace those FromQuery on one object from body?

Reply:
> Maybe replace those FromQuery on one object from body?

That's a get request, it's said that sending GET requests with body is not a good practice